mlflow/h2o.py

```diff
@@ -111,8 +111,8 @@ def log_model(h2o_model, artifact_path, conda_env=None, **kwargs):
     :param conda_env: Either a dictionary representation of a Conda environment or the path to a
                       Conda environment yaml file. If provided, this decribes the environment
                       this model should be run in. At minimum, it should specify the dependencies
-                      contained in :func:`get_default_conda_env()`. If `None`, the default
-                      :func:`get_default_conda_env()` environment is added to the model.
+                      contained in ``mlflow.h2o.DEFAULT_CONDA_ENV``. If ``None``, the default
```

Comment:
```
``mlflow.h2o.DEFAULT_CONDA_ENV``
```
should be changed to
```
 :func:`get_default_conda_env()`
```
here and on line 115.

Reply:
These are merge boo-boos. Will fix.